### PR TITLE
Add `import` to new AdminTool.

### PR DESCRIPTION
**Pull request by srbaker**

### CODE REVIEW

File: community/command-line/pom.xml [hunk 1]
```diff
@@ -66,6 +66,25 @@
...
+      <groupId>org.neo4j</groupId>
+      <artifactId>neo4j-kernel</artifactId>
+      <version>${project.version}</version>
+      <scope>test</scope>
+      <type>test-jar</type>
+    </dependency>
+    <dependency>
+      <groupId>org.neo4j</groupId>
+      <artifactId>neo4j-dbms</artifactId>
+      <version>3.1.0-SNAPSHOT</version>
```
Comment (spacecowboy) [inline community/command-line/pom.xml]: This should also be `${project.version}` ?

File: community/command-line/src/test/java/org/neo4j/commandline/admin/ImportCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+        {
+            assertThat( e.getMessage(), containsString( "does not contain a database" ) );
+        }
+    }
+
+    @Ignore
+    public void copiesDatabaseFromOldLocationToNewLocation() {}
+
+    @Ignore
+    public void removesOldMessagesLog() {}
```
Comment (benbc) [inline community/command-line/src/test/java/org/neo4j/commandline/admin/ImportCommandTest.java]: @srbaker Tests to complete here.

**Comment by benbc** [comment]
@srbaker You can delete even more code than you think.

`neo4j-admin` doesn't need to do any argument checking at all. Just pass everything to Java. Then you can move the "is there even a command" check into Java and you can get rid of the `bad-usage` hack. You can inline the `pass_through` function.

**Comment by chrisvest** [comment]
From the name I thought this was an admin-tool'ification of import-tool. Perhaps a different name will avoid similar confusion for users?

**Comment by srbaker** [comment]
@chrisvest This is simply replacing `neo4j-admin import` which was previously implemented in bash, with an equivalent in Java. I agree that the name is confusing, but it's been that way for a while already.

**Comment by benbc** [comment]
This is why it has a `--mode` flag. The idea is to pull `neo4j-import` into this as `neo4j-admin import --mode=csv`.

**Pull request merged**